gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/checkpoint

```diff
@@ -0,0 +1,16 @@
+model_checkpoint_path: "ckpt-7"
+all_model_checkpoint_paths: "ckpt-1"
+all_model_checkpoint_paths: "ckpt-2"
+all_model_checkpoint_paths: "ckpt-3"
+all_model_checkpoint_paths: "ckpt-4"
+all_model_checkpoint_paths: "ckpt-5"
+all_model_checkpoint_paths: "ckpt-6"
+all_model_checkpoint_paths: "ckpt-7"
+all_model_checkpoint_timestamps: 1611132306.153513
+all_model_checkpoint_timestamps: 1611132921.8376176
+all_model_checkpoint_timestamps: 1611133507.366148
+all_model_checkpoint_timestamps: 1611134092.1264625
+all_model_checkpoint_timestamps: 1611134677.455968
+all_model_checkpoint_timestamps: 1611135262.102583
+all_model_checkpoint_timestamps: 1611135849.2558124
+last_preserved_timestamp: 1610998242.017333
```

Comment:
I'm not sure I understand what is this @BaniaFonseca. Is this supposed to be in the repo?

Reply:
No

Reply:
Ah, then you probably to need to run `git rm` on this file, and add `checkpoint` to gitignore. I think the gitignore only has the *folder* checkpoint, rather than the file.